Packs/SplunkPy/Playbooks/playbook-Splunk_Generic.yml

```diff
@@ -52,7 +52,7 @@ inputs:
   playbookInputQuery:
   required: false
   value:
-    simple: 'Adjustment/Exclusion for offense '
+    simple: 'Adjustment/Exclusion for notable '
```

Comment:
is the space at the end supposed to be here or is that a bug?

Reply:
Supposed